projects/addon-doc/src/components/language-switcher/language-switcher.component.html

```diff
@@ -10,7 +10,7 @@
             <img
                 alt=""
                 class="t-flag"
-                [src]="getFlagPath(flags.get(name))"
+                [src]="flags.get(name)! | tuiFlag"
```

Comment:
what do you think?

`transform(countryIsoCode?: TuiCountryIsoCode | string | null): string | null {`

Reply:
I suppose it would be easier to use.